assets/cubyz/biomes/forest/base.zig.zon

```diff
@@ -14,80 +14,200 @@
 	},
 	.structures = .{
 		.{
-			.id = "cubyz:simple_tree",
-			.leaves = "cubyz:oak_leaves",
-			.log = "cubyz:oak_log",
-			.top = "cubyz:oak_top",
-			.chance = 0.4,
-			.type = .round,
-			.height = 6,
-			.height_variation = 3,
-			.leafElongation = 1.0,
-			.deltaLeafElongation = 0.2,
-		},
-		.{
-			.id = "cubyz:simple_tree",
-			.leaves = "cubyz:birch_leaves",
-			.log = "cubyz:birch_log",
-			.top = "cubyz:birch_top",
-			.chance = 0.2,
-			.type = .round,
-			.height = 10,
-			.height_variation = 5,
-			.leafRadius = 2.5,
-			.leafRadius_variation = 1.5,
-			.leafElongation = 1.7,
-			.deltaLeafElongation = 0.2,
-		},
-		.{
-			.id = "cubyz:fallen_tree",
-			.log = "cubyz:oak_log",
-			.top = "cubyz:oak_log",
+			.id = "cubyz:tree",
 			.chance = 0.005,
-			.height = 6,
-			.height_variation = 3,
+			.heightOffset = -1,
+			.segments = .{
+				.{
+					.height = 3,
+					.stemThickness = 2,
+					.mushroomChance = 0.05,
+					.blocks = .{
+						.wood = "cubyz:oak_log",
+						.mushroom = "cubyz:bolete",
+					},
+				},
+				.{
+					.height = 3,
+					.stemThickness = 2,
+					.branchChance = 0.1,
+					.branchSpacing = 1,
+					.branchSpawnMode = .alternating_spaced,
+					.branchMaxCount = 3,
+					.leavesChance = 0.3,
+					.blocks = .{
+						.leaves = "cubyz:oak_leaves",
+						.wood = "cubyz:oak_log",
+						.branch = "cubyz:oak_branch",
+					},
+					.branchSegmentSeriesVariants = .{
+						.{.forward = .{}},
+						.{.left = .{}},
+						.{.right = .{}},
+						.{.forward = .{}, .left = .{}},
+						.{.forward = .{}, .right = .{}},
+						.{.forward = .{}, .right = .{}, .left = .{}},
+					},
+				},
+				.{
+					.height = 6,
+					.stemThickness = 2,
+					.branchChance = 0.25,
+					.branchSpacing = 1,
+					.branchSpawnMode = .alternating_spaced,
+					.branchMaxCount = 18,
+					.branchMaxCountPerLevel = 2,
+					.leavesChance = 0.0,
+					.leavesBlobChance = 1.0,
+					.leavesBlobRadius = 2.3,
+					.leavesBlobRadiusDelta = 0.4,
+					.blocks = .{
+						.leaves = "cubyz:oak_leaves",
+						.wood = "cubyz:oak_log",
+						.branch = "cubyz:oak_branch",
+					},
+					.branchSegmentSeriesVariants = .{.{
```

Comment:
Formatting our formatter produces for that piece of code it awkward to say at least :C

Reply:
It splits lines based on whether the struct has trailing commas or not.

Reply:
Oh, so more commas it is then.